### PR TITLE
fastlane for F-Droid

### PR DESCRIPTION
**Pull request by linsui**
I'm not familiar with Traditional Chinese so I didn't add zh-TW locale. I thought it makes sense since this is a Taiwanese learning app. It would be great if you can help. Thanks!

### CODE REVIEW

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+A Taiwanese (tâi-gí) learning app inspired by moedict & quizlet
+
+<h3>Functions</h3>
+
+<ul>
+<li>Learning new words</li>
+<li>Watching video and learning related words</li>
+<li>Reviewing words with flaping card game</li>
```
Comment (hschueh) [inline fastlane/metadata/android/en-US/full_description.txt]: typo: `flipping cards` or `card flipping`

**Review by hschueh — CHANGES_REQUESTED** [review]
One small request 👍  I guess you mean I can add the zh-TW by myself?

**Comment by linsui** [comment]
Yes! I'm not familiar that so I can't do that myself.

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (linsui) [reply fastlane/metadata/android/en-US/full_description.txt]: Fixed!

**Review by hschueh — APPROVED** [review]
LGTM!

**Comment by hschueh** [comment]
> Yes! I'm not familiar that so I can't do that myself.

Done!

**Comment by linsui** [comment]
Thanks!

**Comment by hschueh** [comment]
> Yes! I'm not familiar that so I can't do that myself.

@linsui Sorry for the question. I found that the copy for zhTW is simplified Chinese on [F-Droid](https://f-droid.org/packages/com.gonnaggstudio.oh_tai_gi/). 
Did I make any mistake with my metadata?

**Comment by linsui** [comment]
It seems it's a bug of the website. In the client the metadata is displayed in the correct locale. I'll report the problem. Thanks!

**Pull request merged**